### PR TITLE
feature/alarm tags

### PR DESCRIPTION
**Pull request by Guslington**
- ability to tag alarms
- document alarm tags

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -20,4 +20,5 @@
 7. [Maintenance Mode](maintenance_mode.md)
 8. [Composite Alarms](composite_alarms.md)
 9. [Alarms for Custom Metrics](custom_metrics.md)
-10. [Dimension Variables](variables.md)
\ No newline at end of file
+10. [Dimension Variables](variables.md)
+10. [Alarm Tags](alarm_tags.md)
```
Comment (jaredbrook) [inline docs/overview.md:24]: 11?

File: lib/cfnguardian/tagger.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      if tags_to_delete.any?
+        logger.debug "Removing tags #{tags_to_delete} from alarm #{alarm_arn}"
+        @client.untag_resource({
+          resource_arn: alarm_arn,
+          tag_keys: tags_to_delete
+        })
+      end
+
+      if tags_changed?(current_tags, new_tags)
+        # Aws::CloudWatch::Errors::Throttling
```
Comment (jaredbrook) [inline lib/cfnguardian/tagger.rb:30]: What's this comment here for?

File: docs/overview.md (hunk 1 above)
Comment (Guslington) [reply docs/overview.md]: fixed up

File: lib/cfnguardian/tagger.rb (hunk 2 above)
Comment (Guslington) [reply lib/cfnguardian/tagger.rb]: it was a reminder to handle throttling errors which is now applied

**Pull request merged**